funnel/models/profile.py

```diff
@@ -16,6 +17,22 @@ class Profile(ProfileBase, db.Model):
 
     description = MarkdownColumn('description', default=u'', nullable=False)
 
+    __roles__ = {
+        'all': {
+            'read': {
+                'id', 'name', 'title', 'description'
+                },
+            },
+        }
+
+    @cached_property
+    def parent_spaces(self):
+        from .space import ProposalSpace
+        spaces_all = ProposalSpace.fetch_sorted().filter(
+            ProposalSpace.profile == self, ProposalSpace.parent_space == None
+        ).all()
+        return spaces_all
```

Comment:
This should be a relationship. Why is it a property?

Reply:
Not sure why it was defined as a property, I just organized it. Anyway, now when I try to define it as a relationship, there is a circular import issue. `ProposalSpace` model requires `Profile` for a relationship and vice versa.

Reply:
You can put the name in quotes. SQLAlchemy will do an `eval` using the model namespace to pick the correct model. That means any valid Python expression will work inside the quotes.